modules/lvms-upgrading-lvms-on-sno.adoc

```diff
@@ -6,17 +6,59 @@
 [id="lvms-upgrading-lvms-on-sno_{context}"]
 = Upgrading {lvms} on {sno} clusters
 
-Currently, it is not possible to upgrade from {rh-storage} Logical Volume Manager Operator 4.11 to {lvms} 4.12 on {sno} clusters.
+You can upgrade the {lvms-first} Operator to align with your {sno} version.
```

Comment:
What does "align with" mean here? Be compatible with? Be supported in combination with? Be equal to?